### PR TITLE
COMP: Fix RTK_LIBRARIES using find_package(RTK)

### PR DESCRIPTION
**Pull request by LucasGandel**
ITK prepends "itk" and appends "-ITK_VERSION_MAJOR.ITK_VERSION_MINOR" to
the RTK TARGET output name. This modifies the library name to itkRTK-5.0
for instance.
When finding package RTK, those prefix/suffix need to be explicitely
added to the target name.

### CODE REVIEW

File: cmake/RTKConfig.cmake.in [hunk 1]
```diff
@@ -23,8 +23,7 @@ endif()
 
 set(RTK_INCLUDE_DIRS "@RTK_INCLUDE_DIRS@")
 set(RTK_LIBRARY_DIRS "@RTK_LIBRARY_DIRS@")
-set(RTK_LIBRARIES "@RTK_LIBRARIES@")
-
+set(RTK_LIBRARIES "itk@RTK_LIBRARIES@-@ITK_VERSION_MAJOR@.@ITK_VERSION_MINOR@")
```
Comment (LucasGandel) [inline cmake/RTKConfig.cmake.in:26]: @hjmjohnson I would rather fix the library name directly in the RTKConfig.cmake file. This way we can keep RTK as the target name inside the project. What do you think?

**Comment by LucasGandel** [comment]
This PR might replace #138. I would prefer changing the library name in the config file only.

File: cmake/RTKConfig.cmake.in (hunk 1 above)
Comment (hjmjohnson) [reply cmake/RTKConfig.cmake.in:26]: This looks good to me.  Have you tested it?

**Comment by LucasGandel** [comment]
@hjmjohnson I have tested it against the FirstReconstruction example. Works well.
@SimonRit I think we can safely merge this.

**Comment by hjmjohnson** [comment]
👍

**Comment by SimonRit** [comment]
Thanks guys for the work.

**Pull request merged**